### PR TITLE
MNT: add maintainers

### PR DESCRIPTION
**Pull request by klauer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by klauer** [comment]
@conda-forge-admin, please rerender

**Review by hhslepicka — APPROVED** [review]

**Comment by klauer** [comment]
@zrylettc, @zllentz, please confirm you're willing to be maintainers.

**Comment by ZryletTC** [comment]
I confirm

**Comment by hhslepicka** [comment]
@conda-forge-admin please rerun-ci

**Comment by hhslepicka** [comment]
@conda-forge-admin, please restart ci

**Comment by hhslepicka** [comment]
We are having a sha256 issue mostly due to versioneer (common thing that happens unfortunately).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: be225c77d5a0941c24bb48e127ee15a9f8a880e134206b38b37ede996c91c7d2
```
Comment (hhslepicka) [inline recipe/meta.yaml]:
```suggestion
  sha256: c2d3c45363c985f40fee0555ba2011301a0cabdf5fa3a8d95bb53b0623e879b8
```

**Review by hhslepicka — CHANGES_REQUESTED** [review]

**Comment by ZLLentz** [comment]
I confirm that I'm willing to be a maintainer

**Pull request merged**